db/migrate/20230703183221_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :user
+      t.text :content
```

Comment:
same over here need DB level constrains. Check all the fields and update the DB level constraints accordingly.